pyrealm/pmodel/acclimation.py

```diff
@@ -222,6 +222,12 @@ def _validate_and_set_datetimes(self, datetimes: NDArray[np.datetime64]) -> None
         if spacing < 0:
             raise ValueError("Datetime sequence must be increasing")
 
+        # Validate that we are dealing indeed with subdaily data
+        if spacing > np.timedelta64(1, "D"):
+            raise ValueError(
+                "The AcclimationModel data for SubdailyPModel is not subdaily."
+            )
+
```

Comment:
Technically, we allow daily data so I think this is:

```suggestion
        # Validate that we are dealing indeed with subdaily data
        if spacing > np.timedelta64(1, "D"):
            raise ValueError(
                "The AcclimationModel requires datetimes sampled at daily frequency or faster."
            )

```